Update exchange list and links

0x3639 opened the pull request:
- Remove exchanges: XeggeX, QuickSwap, SpiritSwap, and Trader Joe
- Improve exchange layout with responsive 3-column grid on desktop
- Update Discord invite link to latest invitation
- Update C# SDK repository URL to hypercore-one organization
- Add Zenon Development Forum link
- Update copyright year to 2025

0x3639 committed:
Update exchange list and links

- Remove exchanges: XeggeX, QuickSwap, SpiritSwap, and Trader Joe
- Improve exchange layout with responsive 3-column grid on desktop
- Update Discord invite link to latest invitation
- Update C# SDK repository URL to hypercore-one organization
- Add Zenon Development Forum link
- Update copyright year to 2025

KingGorrin reviewed (approved):
Looks good. Maybe you can make the year dynamic with javascript; otherwise you have to change it again next year.

0x3639 committed:
Make copyright year dynamic

Update footer copyright to automatically display current year using JavaScript. This eliminates the need for manual yearly updates.

0x3639 commented:
> Looks good. Maybe you can make the year dynamic with javascript; otherwise you have to change it again next year.

Good idea.  Just made that change.

digitalSloth commented on css/styles.css:
Does this need to be minified? Normally this would be done as part of a build process so we'd keep the un-minified version and a `.min` copy would be created automatically

0x3639 replied:
Good idea.  Maybe we should handle that with a separate PR.  We have the Image optimization PR and then I've got a lazy load PR next.  After those get approved I can set a new build process or look into another approach to minify .css and .js.  I have a few more website optimizations in mind, but the previous feedback was to handle each task with separate PRs.   I previously did one large PR and it was difficult to review everything.

digitalSloth replied:
Yeah I think it would be better to deal with the minification as a separate PR, maybe for now leave it un-minified and just update the exchange links

0x3639 replied:
Fixed.  Wanted to point out this PR improves how the columns are rendered for the exchanges so we can get 1, 2 and 3 columns based on screen width.

digitalSloth commented on index.html:
```diff
@@ -505,23 +505,6 @@ <h3>Uniswap</h3>
               </div>
-              <div class="trade__column">
-                <div class="trade__card">
-                  <img src="./img/xeggex.svg" alt="XeggeX" />
```
Maybe worth deleting any images which are no longer needed

0x3639 replied:
Good idea.  Done

0x3639 committed:
Remove unused exchange images

Delete image files for exchanges no longer listed on the site:
- XeggeX
- QuickSwap
- SpiritSwap
- Trader Joe
- STEX (3 files: card, icon, icon-black)

Also remove unused icon variants:
- 1inch-icon.svg (using card version)
- pancakeswap-icon.svg (using card version)
- rango-icon.svg (exchange not listed)

Total: 10 unused image files removed

0x3639 committed:
rebuilt CSS from SASS source using expanded style instead of minified output
